fe/fe-core/src/main/java/org/apache/doris/job/common/JobStatus.java

```diff
@@ -19,7 +19,10 @@
 
 public enum JobStatus {
 
-
+    /**
+     * For streaming job, if the task has not been created, the job will be in pending state.
+     */
+    PENDING,
```

Comment:
I am not sure if there a compability issues add an enum at the front.

Reply:
There is no compability issues